deepmd/pt/model/model/make_model.py

```
@@ -293,6 +296,15 @@ def forward_common_lower(
                 aparam=ap,
                 comm_dict=comm_dict,
             )
+            # add weight to atomic_output
+            if hasattr(self.atomic_model, "fitting_net"):
+                if hasattr(self.atomic_model.fitting_net, "var_name"):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)